docs/UserGuide.md

```diff
@@ -3,271 +3,443 @@ layout: page
-* Without the use of delimiter `&`, all keywords following the `FLAG` will be used in the search. e.g. `Aaron Tan` will only return persons with `Aaron Tan` in their names.
-* Clients whose chosen field contains at least one keyword will be returned (i.e. OR search). e.g. Hans & Bo will return Hans Sum, Bo Yang.
-* Filter options are set by typing the attribute name after a dash. e.g. `-phone` filters phone number information of the listed clients
+* The delimiter `&` between keywords is used to search using multiple keywords. 
+  * E.g. `Aaron & Tan` will return all persons with either `Aaron` or `Tan` in their names.
+* Without the use of delimiter `&`, all keywords following the `FLAG` will be used in the search. 
+  * E.g. `Aaron Tan` will only return persons with `Aaron Tan` in their names.
+* Clients with matching keywords in the chosen field will be returned (i.e. OR search). 
+  * E.g. `Hans & Bo` will return person `Hans Sum` and `Bo Yang`.
+* Filter options are set by typing the attribute name after a dash.
+  * E.g. `-phone` filters phone number information of the listed clients.
+    
+**Examples**:
+* Find `Alex David` and `Alex Yeoh`.
+  * `find n/alex`
 
+  ![find alex](images/find-alex.png)
 
-Examples:
-* `find n/John` returns `john` and `John Doe`
-* `find n/alex david` returns `Alex David`
-* `find n/alex & david` returns `Alex Yeoh`, `David Li`
-* `find a/orchard` returns `Alex Yeoh` whose address is `Orchard Road`<br>
-* `find n/alex & david -address -phone` returns `Alex Yeoh`, `David Li` and only shows their phone numbers and addresses
+* Find `Alex David`.
+  * `find n/alex david`
+  
+  ![find_alex_david](images/find_alex_david.png)
+  
+* Find `Alex David`, `Alex Yeoh` and `David Li`.
+  * `find n/alex & david`
+  
+  ![find_alex_&_david](images/find_alex_&_david.png)
 
-### Deleting a person : `delete`
+* Find all persons whose address contains `geylang`.
+  * `find a/geylang` - returns `Alex Yeoh` whose address is `Blk 30 Geylang Street 29, #06-40`
 
-Deletes the specified client from ClientBook after listing or finding contacts.
+    ![find by address](images/find-by-address.png)
 
-Format: `delete INDEX`
 
-* Deletes the client at the specified `INDEX`.
+### Display policies associated with selected client: `policy`
+
+**Purpose**: Launches a popup window to show all the policies associated with the selected contact, if the selected contact has any policies.
+
+![without policy URL](images/without_policy_URL.png)
+
+**Format**: `policy INDEX`
+
+* Selects the client at the specified `INDEX`.
 * The index refers to the index number shown in the displayed client list.
 * The index **must be a positive integer** 1, 2, 3, …​
+* Policy ID cannot contain the `>` character!
+* Note that your URLs with should not contain `>` characters either! They are not valid URLs by the Internet's definition.
```

Comment:
I am not sure if these 2 lines should be in this policy command section. Feels like it should be stated in the add/edit command section where the policies are added/edited.

Reply:
Hmm okay, I think I will be adding a section to talk about the fields that can be added for each contact.

Reply:
Yeap, that works too!